src/vmm/tests/integration_tests.rs

```diff
@@ -312,7 +312,7 @@ fn verify_create_snapshot(is_diff: bool) -> (TempFile, TempFile) {
                 snapshot_type,
                 snapshot_path: snapshot_file.as_path().to_path_buf(),
                 mem_file_path: memory_file.as_path().to_path_buf(),
-                version: Some(String::from("0.23.0")),
+                version: Some(String::from("0.24.0")),
```

Comment:
This only validates that we can create snapshots for current version. I would think if it is worth creating a snapshot for current aand all previous version.

Reply:
We have an integration test checking for this.